tests/test_clos/test_workflow_id.py

```diff
@@ -0,0 +1,56 @@
+"""pytest is added for checking workflow_id"""
```

Comment:
```suggestion
"""pytest is added for checking Job.workflow_id"""
```
No really needed though.